journal-writings/modernized/07-chapter-5.adoc

```diff
@@ -79,8 +78,7 @@ These committees meeting together, had a weighty conference in the fear of the L
 at which time,
 I perceived there were many Friends under a scruple like that before-mentioned.^
 footnote:[Christians refused to pay taxes to support Heathen temples.
-See Primitive Christianity, part III.
-page 327]
+See Primitive Christianity, part III. page 327]
```

Comment:
I'll add this to the styleguide soon, but I want to start correctly designating book titles we find, like this.  The asciidoc syntax is below (which wraps a span with a class around some words):

```ASCIIDOC
See [.book-title]#Primitive Christianity#, part III. page 327
```

So, it's a standard looking class-designation `[.book-title]`, but because it's not on a whole paragraph or block, you use `#` to mark the beginning and end. 🔬 💻